app/Providers/AppServiceProvider.php

```diff
@@ -55,6 +56,28 @@ public function boot(MeasurementManager $measure)
         $this->app->booted('\LibreNMS\Config::load');
         $this->app->booted('\App\Http\Controllers\Auth\SocialiteController::registerEventListeners');
 
+        // set up BugSnag
+        $this->app->booted(function () {
+            if (Config::get('reporting.error')) {
+                /** @var \Bugsnag\Client $bugsnag */
+                $reporting_key = '1df714c933472bdf666fc50b2a6a5d98';
```

Comment:
why set this here and not in config/bugsnag.php ?

Reply:
I didn't see an on/off value for bugsnag.  If we set this in .env reporting will be enabled before we bring up the database and check if `reporting.error` is enabled.